backend/games/wordGuesser/ClassicWordle.py

```diff
@@ -0,0 +1,33 @@
+from typing import List
+
+
+class ClassicWordle:
+    # maximum number of guesses
+    MAX_GUESSES = 6
+    # represents the state of the game, index 0 will hold the "goal word"
+    gameBoard = List[str]
```

Comment:
FYI `gameBoard` is a class-variable, not and instance-variable, which means all objects of this class will mutate the same variable. We want to be able to host multiple games at the same time, so this should be changed to an instance variable.

More details here: https://www.tutorialsteacher.com/articles/class-attributes-vs-instance-attributes-in-python

`MAX_GUESSES` is fine as a class variable.